etc/ATSCam/primary_hdu.header

```diff
@@ -30,13 +30,14 @@ OBSGEO-Y=          -5208470.95 / [m] Y-axis Geocentric coordinate
 OBSGEO-Z=          -3195172.08 / [m] Z-axis Geocentric coordinate
 
         ---- Pointing info, etc. ----
-RATEL   =                      / Telescope RA of observation
-DECTEL  =                      / Telescope DEC of observation
-ROTPATEL=                      / Telescope Rotation
-ROTCOORD= 'sky'                / Telescope Rotation Coordinates
-RA      =                      / RA of Target
-DEC     =                      / DEC of Target
+RA      =                      / Telescope RA 
+DEC     =                      / Telescope DEC
+RASTART =                      / RA of Target at start
```

Comment:
It is important to say these are calculated from the telelescope Az/El position!

Reply:
Comment added to the header template.